src/expression.h

```diff
@@ -1112,7 +1114,7 @@ class IndexExp : public BinExp
 public:
     VarDeclaration *lengthVar;
     bool modifiable;
-    bool skipboundscheck;
+    bool indexInRange;
```

Comment:
First I had tried to think names for the new two fields in `SliceExp` like `skipXXX`, but I couldn't do it.
Then I changed the name `IndexExp::skipboundscheck` to `indexIsInBounds` to make the names consistent between `IndexExp` and `SliceExp`.